sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Client/CryptographyClient.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Client
+{
+    using Azure.Core;
+    using Azure.Core.Pipeline;
+    using Azure.Security.KeyVault.Cryptography.Base;
+    using Azure.Security.KeyVault.Cryptography.CryptoProvider;
+    using Azure.Security.KeyVault.Cryptography.Interface;
+    using Azure.Security.KeyVault.Cryptography.Utilities;
+    using Azure.Security.KeyVault.Keys;
+    using System;
+
+    /// <summary>
+    /// The Cryptography Client 
+    /// </summary>
+    public class CryptographyClient
+    {
+        #region const
+        const string DEFAULT_KV_SCOPE_URI = @"https://vault.azure.net/.default";
+        #endregion
+
+        #region fields
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public TokenCredential Credentials { get; }
+        /// <summary>
+        /// Cryptography Client Options
+        /// </summary>
+        public CryptographyClientOptions CryptoClientOptions { get; }
+
+        /// <summary>
+        /// HttpPipeline
+        /// </summary>
+        public HttpPipeline Pipeline {get;}
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public ICryptographyProvider CryptoProvider { get; protected set; }
+
+        /// <summary>
+        /// KeyVault Uri
+        /// </summary>
+        public Uri KeyVaultKeyUri { get; }
+
+        /// <summary>
+        /// KeyVault Key
+        /// </summary>
+        public Key KeyVaultKey { get; }
```

Comment:
This shouldn't be a property because we are lazily fetching the key this is what the GetKey and GetKeyAsync methods on the strawman are for.

Reply:
Any reason we do not want these properties, especially for data that is being passed to the client at constructor time?
Agreed there are constcutors that do not pass Key in all overloads, but do you see any harm to expose the Key that becomes part of the client (regardless of fetched or passed in during construction)